app/views/gyms/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Lista de Academias</h1>

```

Comment:
O que acha de colocar um pouco mais de informação nessa view, tipo:

```
Endereço
O endereço da academia aqui
```

Reply:
Feito, mas vou melhorar quando for colocar o estilo na página.

Reply:
Você vai fazer depois em outro pull request ou nesse mesmo?